frontend/src/component/ui/ChangeLogModal.scss

```diff
@@ -0,0 +1,113 @@
+@import '../styles/color';
+@import '../styles/corner';
+@import '../styles/scrollbar';
+
+.modal-wrapper {
+  left: 0%;
+  position: fixed;
+  top: 0%;
+  width: 100%;
+  height: 100%;
+
+  .modal-body {
+    background-color: $color-white;
+    border: 1px solid $color-light-gray-1;
+    left: 50%;
+    position: fixed;
+    top: 50%;
+    width: 500px;
+    height: 500px;
+    transform: translateX(-50%) translateY(-50%);
+    @include round-corner;
+
+    .modal-header {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      font-size: 20px;
+      line-height: 1;
+      text-align: left;
+      padding: 20px;
+      border-bottom: 1px solid $color-light-gray-1;
+      font-weight: 600;
+
+      i.clear {
+        cursor: pointer;
+      }
+    }
+
+    ul {
+      height: calc(500px - 114px);
+      list-style: none;
+      display: block;
+      padding: 0;
+      margin: 0;
+      overflow-y: scroll;
+      background-color: $color-white-smoke;
```

Comment:
```suggestion
```

Reply:
Why remove background from ul? there is background color in [this image](https://user-images.githubusercontent.com/3537801/69011841-ae3d9c00-0923-11ea-9821-24ea5e0318c6.png)

Reply:
Oh, I misunderstand.